blockchain/common_test.go

```diff
@@ -525,6 +525,54 @@ func findDeploymentAllYesChoices(params *chaincfg.Params, voteIDs []string) (uin
 		"provided vote IDs")
 }
 
+// mergeAgendas moves all the specified agendas into the same deployment,
+// suitable for being voted at the same time.
+//
+// The agendas will be moved to the deployment that contains the first agenda
+// in the voteIDs slice.
+//
+// Returns the resulting deployment version.
+func mergeAgendas(params *chaincfg.Params, voteIDs []string) (uint32, error) {
```

Comment:
Nice addition!  This prevents the need to go through two voting cycles in the tests saving a lot of time.